src/webScience.js

```diff
@@ -53,3 +53,6 @@ export { linkExposure }
 
 import * as socialMediaLinkSharing from "./socialMediaLinkSharing.js"
 export { socialMediaLinkSharing }
+
+import * as randomization from "./randomization.js"
+export { randomization }
```

Comment:
I wonder if we could do these as one line like: `export { randomization } from './randomization.js`? I haven't tried it yet but remember [seeing it on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#re-exporting_aggregating).

Reply:
I looked into this. There's a pending proposal for namespace re-exports: https://github.com/tc39/proposal-export-ns-from

Reply:
Hm, wasn't that merged in https://github.com/tc39/ecma262/pull/1174 ? Although looking at the compatibility table on MDN we should probably avoid it for now anyway.

Reply:
Ah, I misunderstood the status of the proposal. Since we already need to use recent browser versions because of various WebExtensions APIs, I think we could switch to this syntax.